README.md

```diff
@@ -2,9 +2,9 @@
 
 A `bfloat16` implementation for projects of [BioVault](https://github.com/biovault) (Biomedical Visual Analytics Unit LUMC - TU Delft)
 
-Originally based upon `dnnl::impl::bfloat16_t` from the [Deep Neural Network Library (DNNL)](https://github.com/intel/mkl-dnn) of Intel Corporation:
```

Comment:
I would keep this documentation in here since it was really originally based on the intel mkl-dnn library